man/conf.py

```diff
@@ -9,7 +10,7 @@
 
 
 def _get_description(fname, base):
-    with file(fname) as f:
+    with contextlib.closing(open(fname)) as f:
```

Comment:
Yes, you're right. I was thinking of sockets which need `contextlib.closing`. Thanks @tchaikov!

Fixed in new commit.